### ISSUE TITLE
Display true or false for each components of a rule in the report or log functionality

### ISSUE THREAD

**Issue by mhungria** [cdisc-org/cdisc-rules-engine#1728]
### Standard

CDISC SDTM/SDTMIG

### Reference Rule ID(s)

All in all standards

### Conformance Rule ID(s) (if published in CORE)

CORE-

### JIRA Ticket

https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-

### Proposed Rule YAML

```YAML
Check:
  all:
    # clause A
    - name: 
      operator: 
      value: 
    # clause B
    - name: 
      operator: 
      value: 
...
```

### Attach any sample test data file(s)

NA

### Expected output

Display true or false for each components of a rule in the report or log functionality.

For example, for the extract yaml segment included, displaying the following in the results or log:

A = true | false, B = true | false. ...

### Any Additional Information

I sort of recall discussing the topic a while, while back with either @SFJohnson24 or @gerrycampion and I got an understanding that the json results included that information.
